### DISCUSSION TITLE
Gatehouse: rather unuseful in vanilla

### DISCUSSION THREAD

**Discussion by GiulioMartini** [Keriew/augustus#302] [Ideas]
I have a suggestion regarding gatehouse buildings.

In their vanilla version they are useful for:

1. stopping common people from exiting the city perimeter
2. providing walls with a nice OPEN door. That's it though! (maybe sentinels can walk on top of them, but can't remember right now)

This brings me to my point: they are rather useless! I mean a gatehouse historically was built for DEFENCE.

Wouldn't it make more sense if they:

1. had a ballista on them, exactly like towers
2. had the ability to shut close when enemies are nearby, with fewer hitpoints than towers or walls (maybe this would be far too complex since it could trigger the "road for Rome is closed" event...)

In exchange for these upgrades, maybe they should employ 6 workers, just like a tower!

**Comment by crudelios** [comment]
Hey, thanks for the suggestion.

Having a ballista on top is an idea. They certainly have the space for that.

The walls shutting is actually a only a visual change. Even though they "look" open, enemies can't really cross them. However it shouldn't be too difficult to add some doors to the gatehouse.
